src/analysis/raster/qgsexiftools.h

```diff
@@ -47,6 +51,13 @@ class ANALYSIS_EXPORT QgsExifTools
      */
     static QgsPoint getGeoTag( const QString &imagePath, bool &ok SIP_OUT );
 
+    /**
+     * Returns TRUE if the image at \a imagePath contains a valid geotag.
+     *
+     * \see getGeoTag()
+     */
+    Q_INVOKABLE static bool hasGeoTag( const QString imagePath );
```

Comment:
```suggestion
    Q_INVOKABLE static bool hasGeoTag( const QString &imagePath );
```

Reply:
@m-kuhn , let me try; one reason why I've added this shortcut function is that QML doesn't support bool &ref. Let's see if it'll be fine with a referenced QString.